mmv1/third_party/terraform/services/dataproc/resource_dataproc_cluster_upgrade_test.go

```diff
@@ -154,59 +154,10 @@ func TestAccDataprocClusterLabelsMigration_withUpdate(t *testing.T) {
 					testAccCheckDataprocClusterExists(t, "google_dataproc_cluster.with_labels", &cluster),
 
 					resource.TestCheckNoResourceAttr("google_dataproc_cluster.with_labels", "labels.%"),
-					// We only provide one, but GCP adds three and goog-dataproc-autozone is added internally, so expect 4.
-					resource.TestCheckResourceAttr("google_dataproc_cluster.with_labels", "effective_labels.%", "4"),
-				),
-			},
-		},
-	})
-}
-
-func TestAccDataprocClusterLabelsMigration_withRemoval(t *testing.T) {
```

Comment:
We can keep the test around for some time, it'll provide a warning if we break an assumption we made in the migration previously.

Reply:
Sounds good.